cpp/tests/structs/utilities_tests.cpp

```diff
@@ -22,7 +22,10 @@
 #include <cudf_test/table_utilities.hpp>
 #include <cudf_test/type_lists.hpp>
 
+#include <cudf/column/column_factories.hpp>
 #include <cudf/detail/aggregation/aggregation.hpp>
+#include <cudf/detail/null_mask.hpp>
+#include <cudf/null_mask.hpp>
 #include <structs/utilities.hpp>
```

Comment:
This header is local to the `src` thus it should come to the first place at top.

```suggestion
#include <structs/utilities.hpp>
...
```